scripts/west_commands/zephyr_ext_common.py

```diff
@@ -52,3 +56,12 @@ def cached_runner_config(build_dir, cache):
                         elf_file, hex_file, bin_file,
                         gdb=gdb, openocd=openocd,
                         openocd_search=openocd_search)
+
+# FIXME we should think of a nicer way to manage sys.path
+# for shared Zephyr code.
+def zephyr_scripts_path():
+    # This relies on this file being zephyr/scripts/foo/bar.py.
+    zephyr_base = Path(os.environ.get('ZEPHYR_BASE',
+                       Path(__file__).parent.parent.parent))
```

Comment:
This will not help if users has done:
```
cd <path-to-projectA>/zephyr
source zephyr-env.sh
cd <path-to-projectB>/zephyr
west build -b<some-board>
```
will start build using a wrong `ZEPHYR_BASE` where the old behavior would have given following warning:
```
WARNING: ZEPHYR_BASE=/<path-to-projectA>/zephyr in the calling environment will be used,           
but the zephyr.base config option in /<path-to-projectB>/zephyr is "zephyr"                   
which implies a different ZEPHYR_BASE=/<path-to-projectB>/zephyr                          
To disable this warning in the future, execute 'west config --global zephyr.base-prefer env'           

```

Reply:
Are you suggesting we should unconditionally use the current zephyr repository?

Reply:
I advocating that we should not change behavior without a prior notice before a release, that we will be doing so.
So in my opinion, we should restore behavior either in https://github.com/zephyrproject-rtos/west/pull/379 , or by having same behavior in Zephyr.
Personally, I would argue that for first, the original behavior is restored in `west`, cause projects that have forked `zephyr` may not get the old behavior restored until their next upmerge.

Reply:
I agree, so why is that a blocker for this PR?

Reply:
@tejlmand can you please revisit your review now that west #379 is merged? I will make sure it's in 0.7.2 whenever we decide to release that.